src/test/java/seedu/address/logic/commands/SortCommandTest.java

```diff
@@ -24,30 +24,30 @@ public class SortCommandTest {
 
     @Test
     public void execute_nameAscendingOrder_success() {
-        executeSuccessfulSortTest(PersonComparator.NAME, true, modelAscendingDateOfLastVisit);
+        executeSuccessfulSortTest(PersonComparator.NAME, true, modelAscendingName);
     }
 
     @Test
     public void execute_nameDescendingOrder_success() {
-        executeSuccessfulSortTest(PersonComparator.NAME, false, modelDescendingDateOfLastVisit);
+        executeSuccessfulSortTest(PersonComparator.NAME, false, modelDescendingName);
     }
 
     @Test
     public void execute_dateOfLastVisitAscending_success() {
-        executeSuccessfulSortTest(PersonComparator.DATE_OF_LAST_VISIT, true, modelDescendingName);
+        executeSuccessfulSortTest(PersonComparator.DATE_OF_LAST_VISIT, true, modelAscendingDateOfLastVisit);
     }
 
     @Test
     public void execute_dateOfLastVisitDescending_success() {
-        executeSuccessfulSortTest(PersonComparator.DATE_OF_LAST_VISIT, false, modelAscendingName);
+        executeSuccessfulSortTest(PersonComparator.DATE_OF_LAST_VISIT, false, modelDescendingDateOfLastVisit);
     }
 
     private void executeSuccessfulSortTest(String sortParameter, boolean isAscending, Model model) {
```

Comment:
Going along with the previous comment we can change the name of the Model input in the parameters for this method to be modelToBeSorted.